src/clusterfuzz/stacktraces/constants.py

```diff
@@ -114,6 +114,8 @@
 FATAL_ERROR_REGEX = re.compile(r'#\s*Fatal error in (.*)')
 FATAL_ERROR_LINE_REGEX = re.compile(r'#\s*Fatal error in (.*), line [0-9]+')
 FATAL_ERROR_UNREACHABLE = re.compile(r'# un(reachable|implemented) code')
+FUZZER_EXIT_REGEX = re.compile(r'^    (?:#0|#1) 0x.*(?:fuzzer|fuzz/|/fuzz)',
```

Comment:
This seems a bit hacky. Are there any other alternatives we can be doing here to extract this in a more reliable way?

Reply:
Thanks Oliver - were you thinking that this might be under/overinclusive? 

The intention was to identify early stacktrace references to fuzzing or a fuzzer (here, frame 0 or 1). I know it looks a bit strange, as this is a heuristic as opposed to something more definite, but applied across all testcases this regex seem to have hit only the right targets. Our analysis of some several hundred fuzzer design/infra-related Android crashes has shown that this kind of exit has been consistently evaluated by our ISE as having ‘No Security Impact’, often with the reasoning that the crash appeared to be within the fuzzer itself or caused by poor fuzzer design. For example,

https://clusterfuzz.corp.google.com/testcase-detail/5379218936561664
https://clusterfuzz.corp.google.com/testcase-detail/6097477842894848
https://clusterfuzz.corp.google.com/testcase-detail/5162986195779584
https://clusterfuzz.corp.google.com/testcase-detail/6489775183101952
https://clusterfuzz.corp.google.com/testcase-detail/5848511005720576

Reply:
My main concern was that this seemed a bit fragile and hacky. If you've done a lot of analysis here and this works for you, LGTM.